internal/cmd/connect/command_custom_plugin_delete.go

```diff
@@ -0,0 +1,52 @@
+package connect
+
+import (
+	"fmt"
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/form"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/resource"
+	"github.com/spf13/cobra"
+)
+
+func (c *customPluginCommand) newDeleteCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:         "delete <id>",
+		Short:       "Delete a custom connector plugin.",
+		Args:        cobra.ExactArgs(1),
```

Comment:
This command will need a rewrite to support multiple deletion after you merge main into this branch, unfortunately. There will be a `README.md` under `pkg/deletion` that explains what to do; you can also look at various other delete.go files for examples.

Reply:
Rewrote this with the new delete